components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt

```diff
@@ -16,6 +16,7 @@ object ErrorPages {
     fun createErrorPage(
         context: Context,
         errorType: ErrorType,
+        uri: String? = "",
```

Comment:
If you're adding a default parameter, then we can assert that the string is non-null. If we _don't_ have it, then it's nullable.

So you can choose to either have `uri: String = ""` or `uri: String?`. You don't need both together :).

Reply:
Sorry, not a Kotlin expert. If I make it `uri: String?` (optional without default value) then I **must** pass a parameter when calling createErrorPage. Can I avoid that without providing a default value?

Reply:
> I must pass a parameter when calling createErrorPage. Can I avoid that without providing a default value?

The default can be null too (`uri: String? = null?`) if that is helpful here.